WorkforceManagement/WorkforceManagement/BangazonDatabaseCreateSeed.sql

```diff
@@ -0,0 +1,405 @@
+    ALTER TABLE dbo.ProductOrder DROP CONSTRAINT FK_OrderProductOrder
+END
+
+IF (OBJECT_ID('dbo.FK_ProductProductOrder', 'F') IS NOT NULL)
+BEGIN
+    ALTER TABLE dbo.ProductOrder DROP CONSTRAINT FK_ProductProductOrder
+END
+
+IF (OBJECT_ID('dbo.FK_CustomerPaymentOrder', 'F') IS NOT NULL)
+BEGIN
+    ALTER TABLE dbo.[Order] DROP CONSTRAINT FK_CustomerPaymentOrder
+END
+
+IF (OBJECT_ID('dbo.FK_CustomerOrder', 'F') IS NOT NULL)
+BEGIN
+    ALTER TABLE dbo.[Order] DROP CONSTRAINT FK_CustomerOrder
+END
+
+IF (OBJECT_ID('dbo.FK_PaymentTypeCustomer', 'F') IS NOT NULL)
+BEGIN
+    ALTER TABLE dbo.CustomerPayment DROP CONSTRAINT FK_PaymentTypeCustomer
+END
+
+IF (OBJECT_ID('dbo.FK_CustomerPayment', 'F') IS NOT NULL)
+BEGIN
+    ALTER TABLE dbo.CustomerPayment DROP CONSTRAINT FK_CustomerPayment
+END
+
+DROP TABLE IF EXISTS Department;
+DROP TABLE IF EXISTS Employee;
+DROP TABLE IF EXISTS EmployeeComputer;
+DROP TABLE IF EXISTS EmployeeTraining;
+DROP TABLE IF EXISTS [Order];
+DROP TABLE IF EXISTS PaymentType;
+DROP TABLE IF EXISTS Product;
+DROP TABLE IF EXISTS ProductOrder;
+DROP TABLE IF EXISTS ProductType;
+DROP TABLE IF EXISTS TrainingProgram;
+DROP TABLE IF EXISTS Computer;
+DROP TABLE IF EXISTS Customer;
+DROP TABLE IF EXISTS CustomerPayment;
+
+
+
+CREATE TABLE Computer (
+  ComputerId          INTEGER NOT NULL PRIMARY KEY IDENTITY,
+  DatePurchased       DATE NOT NULL,
+  DateDecommissioned  DATE,
+  Working             BIT NOT NULL,
+  ModelName           varchar(80) NOT NULL,
+  Manufacturer        varchar(80) NOT NULL
+);
+CREATE TABLE TrainingProgram (
+  TrainingProgramId   INTEGER NOT NULL PRIMARY KEY IDENTITY,
+  ProgramName         varchar(80) NOT NULL,
+  StartDate           DATE NOT NULL,
+  EndDate             DATE NOT NULL,
+  MaximumAttendees    INTEGER NOT NULL
+);
+CREATE TABLE Department (
+  DepartmentId        INTEGER NOT NULL PRIMARY KEY IDENTITY,
+  DepartmentName      varchar(80) NOT NULL,
+  ExpenseBudget       INTEGER NOT NULL
+);
+CREATE TABLE Employee (
+  EmployeeId          INTEGER NOT NULL PRIMARY KEY IDENTITY,
+  FirstName           varchar(30) NOT NULL,
+  LastName            varchar(30) NOT NULL,
+  Email               varchar(50) NOT NULL,
+  Supervisor          BIT NOT NULL,
+  StartDate           DATE NOT NULL,
+  EndDate			  DATE,
+  DepartmentId        INTEGER NOT NULL,
+  CONSTRAINT FK_DepartmentEmployee FOREIGN KEY(DepartmentId)REFERENCES Department(DepartmentId)
+);
+CREATE TABLE EmployeeTraining (
+  EmployeeTrainingId  INTEGER NOT NULL PRIMARY KEY IDENTITY,
+  TrainingProgramId   INTEGER NOT NULL,
+  EmployeeId          INTEGER NOT NULL,
+  CONSTRAINT FK_TrainingProgram FOREIGN KEY(TrainingProgramId)REFERENCES TrainingProgram(TrainingProgramId),
+  CONSTRAINT FK_EmployeeTraining FOREIGN KEY(EmployeeId)REFERENCES Employee(EmployeeId)
+);
+CREATE TABLE EmployeeComputer (
+  EmployeeComputerId  INTEGER NOT NULL PRIMARY KEY IDENTITY,
+  DateAssigned        DATE NOT NULL,
+  DateReturned        DATE NOT NULL,

```

Comment:
Please change DateReturned from DATE NOT NULL to DATE.

Reply:
It is finished